### DISCUSSION TITLE
Why Tracking

### DISCUSSION THREAD

**Discussion by Xandorion** [KRTirtho/spotube#256] [Q&A]
Hey, i just saw that duckduckgo blocks some Trackers of Spotube, could u explain why it needs these? Sorry that its German, if u need one in English i can try to make one.

![Screenshot_20221004-164126_DuckDuckGo](https://user-images.githubusercontent.com/96028955/193850447-20c999eb-8542-48fe-9867-536dd1209da1.jpg)

**Comment by KRTirtho** [comment]
It must be a false positive. Spotube doesn't have any kind of dependencies that tracks user or sends data to Google
That's the whole point of Spotube no data collection. Not even analytical data/crash logs

>  if u need one in English i can try to make one.

Please provide. Not everyone understands German :smile:

**Reply by Xandorion** [reply]
Sorry, the Duckduckgo app itself has now language option, but could change it with language of the device. Here again one in english, sorry.

I understand what u said about it but, why does it says these things? This DuckDuckGo tracker normally works very well.

![Screenshot_20221005-093408_DuckDuckGo](https://user-images.githubusercontent.com/96028955/194005850-6a9e4264-51c2-4037-b369-1fe1bcc206fe.jpg)

**Reply by onkq** [reply]
I can see you're very new to this world. 

The thing you are seeing is false positives as Kingkor said, and, since YouTube and the connected services needed for the function of the software has companies willing to abuse you, duckduckgo goes after the domain name you're connected to, and has no way of knowing whether you're using Spotube. But it also seems like you don't quite understand what you are doing. Duckduckgo is horrible and unreliable!, does some tracking themselves. I wouldn't use duckduckgo for anything else other than searching. It's not a non-profit organization nor open source, so who knows what else they do in the background. I can recommend you take a closer look at https://privacyguides.org to read and learn.

Just wanna add to the last part, don't give your trust to https://privacyguides.org/ all together, they have some shady people in the project, but what they write is correct and true, and they do the research (mostly). There are also bias opinions from the lead maintainers
